wandb/data_types.py

```diff
@@ -68,6 +68,7 @@
     )
 
 __all__ = [
+    "Audio",
```

Comment:
I'm not sure if adding this here is okay?

Reply:
Nice catch, yes, this should be here